package.json

```diff
@@ -64,16 +68,24 @@
   },
   "devDependencies": {
     "@babel/core": "7.21.3",
+    "@react-native-community/datetimepicker": "7.6.1",
+    "@react-native-community/slider": "4.4.2",
+    "@storybook/addon-ondevice-actions": "7.6.14",
+    "@storybook/addon-ondevice-controls": "7.6.14",
+    "@storybook/react-native": "7.6.14",
     "@trivago/prettier-plugin-sort-imports": "4.3.0",
     "@types/lodash": "4.14.202",
     "@types/react": "18.2.45",
     "@typescript-eslint/eslint-plugin": "5.59.5",
+    "babel-loader": "8.3.0",
+    "cross-env": "7.0.3",
```

Comment:
I think we should not have this (you add it because of your activate storybook script that I think we should remove)